CMakeLists.txt

```diff
@@ -43,11 +43,7 @@ include(LXQtPreventInSourceBuilds)
 include(LXQtCompilerSettings NO_POLICY_SCOPE)
 
 # Deal with a bug in polkit-qt5
```

Comment:
@stefonarch 
WOuld you remove the above comment too?

Reply:
Done